packages/@react-spectrum/table/src/InsertionIndicator.tsx

```diff
@@ -0,0 +1,50 @@
+import {classNames} from '@react-spectrum/utils';
+import {ItemDropTarget} from '@react-types/shared';
+import React, {useRef} from 'react';
+import styles from './table.css';
+import {useTableContext} from './TableView';
+import {useVisuallyHidden} from '@react-aria/visually-hidden';
+
+interface InsertionIndicatorProps {
+  target: ItemDropTarget,
+  rowProps: any
+}
+
+export default function InsertionIndicator(props: InsertionIndicatorProps) {
+  let {dropState, dragAndDropHooks} = useTableContext();
+  const {target, rowProps} = props;
+
+  let ref = useRef();
+  let {dropIndicatorProps} = dragAndDropHooks.useDropIndicator(props, dropState, ref);
+  let {visuallyHiddenProps} = useVisuallyHidden();
+
+  let isDropTarget = dropState.isDropTarget(target);
+
+  if (!isDropTarget && dropIndicatorProps['aria-hidden']) {
+    return null;
+  }
+
+  let style = {
```

Comment:
I don't see this in react-spectrum/list/InsertionIndicator. Why is it needed here? Not that it's wrong, just surprised.

Reply:
They were all rendering at the top of the container, so I positioned them absolute. I think becase the tableview rows themselves are positioned differently than listview items. See https://github.com/adobe/react-spectrum/pull/4008#discussion_r1109086970 for a related answer.